src/lib/util/environment.ts

```diff
@@ -12,15 +11,18 @@ export const DEBUG = envParseBoolean('DEBUG', true);
 export const APP_ENV = envParseString('APP_ENV');
 
 // GENERIC
-export const { CLIENT_NAME, BOT_AVATAR } = process.env;
+export const CLIENT_NAME = envParseString('CLIENT_NAME', 'Birthdayy');
+export const CLIENT_AVATAR = envParseString(
+	'CLIENT_AVATAR',
+	'https://cdn.discordapp.com/avatars/1039089174948626473/8d7f1e8a8b9f6e2b2f4b9a2c9b2b3a7e.webp'
```

Comment:
This image is empty is this the expected result?

Reply:
my bad, we should use the bot avatar via code not hardcoded, it would be better I think

Reply:
Yes, I think it would be better 👍